src/sympc/config/config.py

```diff
@@ -7,18 +7,25 @@
 
 @dataclass
 class Config:
-    """Config is a class used inside a Session (see Session) that specifies
-    diffenret options that can be used for the Fixed Precision Encoder.
+    """Session configuration.
 
-    Arguments:
-        encoder_base (int): the base for the encoder
-        encoder_precision (int): the precision for the encoder
+    Attributes:
+        encoder_base (int): Base for the encoder.
+        encoder_precision: Precision for the encoder.
     """
 
     encoder_precision: int = field()
     encoder_base: int = field()
 
     def __init__(self, encoder_base: int = 2, encoder_precision: int = 16) -> None:
-        """Initializer for the Config."""
+        """Session configuration.
+
+        Config can be used inside a Session (see Session) to
+        specify different options that can be used for the Fixed Precision Encoder.
+
+        Args:
```

Comment:
Q: Should we have ```Args``` or ```Attributes``` or ```Arguments```?

Reply:
`Attributes` key is used for the attributes of a class.
`Args` key is used for arguments of a function or method from from a class
`Arguments` is not useed in Google format (as far as I know): see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html